src/users/users.test.ts

```diff
@@ -102,286 +39,584 @@ const ctx: IContext = {
   }),
 };
 
-test('should show the users pages', async t => {
-  const response = await users.listUsers(ctx, {
-    organizationGUID: '3deb9f04-b449-4f94-b3dd-c73cefe5b275',
+function composeOrgRoles(setup: object) {
+  const defaultRoles = {
+    billing_managers: {
+      current: '0',
+    },
+    managers: {
+      current: '0',
+    },
+    auditors: {
+      current: '0',
+    },
+  };
+
+  return {
+    ...defaultRoles,
+    ...setup,
+  };
+}
+
+function composeSpaceRoles(setup: object) {
+  const defaultRoles = {
+    developers: {
+      current: '0',
+    },
+    managers: {
+      current: '0',
+    },
+    auditors: {
+      current: '0',
+    },
+  };
+
+  return {
+    ...defaultRoles,
+    ...setup,
+  };
+}
+
+test('ordinary set of tests', async _t => {
```

Comment:
Is this an odd title?

Reply:
It is xD Naming is hard and I couldn't come up with a better title for "current and old style of tests, which work but are somewhat ugly"